Platform/DispatchQueue+Extensions.swift

```diff
@@ -7,15 +7,10 @@
 //
 
 import Dispatch
+import Foundation
 
 extension DispatchQueue {
-    private static var token: DispatchSpecificKey<()> = {
-        let key = DispatchSpecificKey<()>()
-        DispatchQueue.main.setSpecific(key: key, value: ())
-        return key
-    }()
-
     static var isMain: Bool {
-        return DispatchQueue.getSpecific(key: token) != nil
+        return Thread.isMainThread
```

Comment:
I believe this entire file should be reverted to develop?

Reply:
Fixed

Reply:
@freak4pc Just confused, can you explain why check by token is better than `Thread.isMainThread`?